pawn/Resources/Callbacks/OnPlayer/OnPlayerCommandText.pwn

```diff
@@ -906,6 +918,42 @@ public OnPlayerCommandText(playerid, cmdtext[]) {
         return 1;
     }
 
+    if (strcmp(cmd, "/vips", true) == 0) {
+        SendClientMessage(playerid, COLOR_ORANGE, "List of VIPs online in Las Venturas Playground");
+
+        new message[128], vipCount = 0, playerLevel[30];
+        for (new player = 0; player <= PlayerManager->highestPlayerId(); player++) {
```

Comment:
I believe calling the variable "playerId" would be more suitable.

"player" would assume that it's the entity player.